### PR TITLE
Add environment

### PR DESCRIPTION
**Pull request by arianna-h-h**

### CODE REVIEW

File: hello.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const http = require('http');
```
Comment (ghaynes1292) [inline hello.js]: server.js

Comment (arianna-h-h) [inline hello.js]: done

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "eslint": "^4.6.1",
+    "eslint-config-airbnb-base": "^12.0.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-jest": "^21.0.2",
+    "jest": "^20.0.4",
+    "standard": "^10.0.3"
+  },
+  "dependencies": {
+    "eslint-plugin-import": "^2.7.0",
+    "import": "0.0.6"
```
Comment (snmgian) [inline package.json]: why do we need this?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  "devDependencies": {
+    "eslint": "^4.6.1",
+    "eslint-config-airbnb-base": "^12.0.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-jest": "^21.0.2",
+    "jest": "^20.0.4",
+    "standard": "^10.0.3"
+  },
+  "dependencies": {
+    "eslint-plugin-import": "^2.7.0",
```
Comment (snmgian) [inline package.json]: Shouldn't be included as a dev dependency ?

**Review by snmgian — CHANGES_REQUESTED** [review]

**Review by snmgian — APPROVED** [review]

**Pull request merged**